cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -50,8 +50,6 @@
  */
 public class BladeCLI implements Runnable {
 
-	public static final File USER_HOME_DIR = new File(System.getProperty("user.home"));
```

Comment:
I don't want to include these changes just yet, I think I am going to refactor how blade gets its extensions directory, I want to change the information flow a bit.  Can you remove the changes to the userHomeDir from this PR and just send the parallel tests changes in a single PR?